tests/system_test.py

```diff
@@ -1204,6 +1204,66 @@ def get_inter_router_conns(self):
         conns = self.management.query(type=CONNECTION_TYPE).get_dicts()
         return [c for c in conns if 'inter-router' in c['role']]
 
+    def get_inter_router_data_conns(self):
```

Comment:
Seems like some of these functions are generic enough to be added to `system_test.py`